app/src/renderer/components/common/TmPage.vue

```diff
@@ -90,6 +93,10 @@ export default {
     refresh: {
       type: Function,
       default: undefined
+    },
+    signInRequired: {
```

Comment:
thinking ahead a little bit (maybe too far ahead) maybe this should be called `addressRequired` because soon you will not need to "sign in" or "be signed in" - you will just have to provide an address.

Reply:
although the address will still be provided on the sign in session

Reply:
I could go either way

Reply:
let's merge it for now.